indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorIOConfig.java

```diff
@@ -62,9 +67,19 @@ public SeekableStreamSupervisorIOConfig(
     this.lateMessageRejectionPeriod = lateMessageRejectionPeriod == null
                                       ? Optional.absent()
                                       : Optional.of(lateMessageRejectionPeriod.toStandardDuration());
+    this.lateMessageRejectionStartDateTime = lateMessageRejectionStartDateTime == null
+                                      ? Optional.absent()
+                                      : Optional.of(lateMessageRejectionStartDateTime);
     this.earlyMessageRejectionPeriod = earlyMessageRejectionPeriod == null
                                        ? Optional.absent()
                                        : Optional.of(earlyMessageRejectionPeriod.toStandardDuration());
+
+    if (this.lateMessageRejectionPeriod.isPresent()
+                && this.lateMessageRejectionStartDateTime.isPresent()) {
+      throw new JsonMappingException("SeekableStreamSupervisorIOConfig does not support "
```

Comment:
for validation errors, we don't use checked exceptions but .. `IAE` would be more appropriate. please remove the `throws Exception` etc from constructor of other classes where it was added due to this.

Reply:
Fixed